### PR TITLE
Polish language added

### PR DESCRIPTION
**Pull request by mateuszgachowski**
Hi,

I have added polish language as it was lacking.

After adding translations I have run `npm run build` and commited compiled JS and CSS files (hope the JS is required too).

Cheers,
Mateusz

### CODE REVIEW

File: sass/_polish.sass [hunk 1]
```diff
@@ -0,0 +1,49 @@
+$upComputer: "Twój komputer jest podłączony do internetu."
+$upDevice: "Twoje urządzenie jest podłączone do internetu."
+$upDeviceSmall: "Twoje urządzenie jest podłączone."
+
+$downComputer: "Twój komputer stracił połączenie z internetem."
+$downDevice: "Twóje urządzenie straciło połączenie z internetem."
+$downDeviceSmall: "Twóje urządzenie nie jest podłączone."
```
Comment (citricacid-pl) [inline sass/_polish.sass]: Should be `Twoje`

File: themes/offline-language-polish.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  }
+}
+/* line 22, ../sass/_content.sass */
+.offline-ui.offline-ui-down .offline-ui-content:before {
+  content: "Twój komputer stracił połączenie z internetem.";
+}
+@media (max-width: 1024px) {
+  /* line 22, ../sass/_content.sass */
+  .offline-ui.offline-ui-down .offline-ui-content:before {
+    content: "Twóje urządzenie straciło połączenie z internetem.";
```
Comment (citricacid-pl) [inline themes/offline-language-polish.css]: `Twoje` again

File: themes/offline-language-polish.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  }
+}
+@media (max-width: 568px) {
+  /* line 22, ../sass/_content.sass */
+  .offline-ui.offline-ui-down .offline-ui-content:before {
+    content: "Twóje urządzenie nie jest podłączone.";
```
Comment (citricacid-pl) [inline themes/offline-language-polish.css]: `Twoje` again

**Comment by citricacid-pl** [comment]
Reviewed the translations and there are few typos.

**Comment by mateuszgachowski** [comment]
@citricacid-pl typos fixed, thanks :)

**Comment by mateuszgachowski** [comment]
Is is possible to merge this one?

**Comment by FranDias** [comment]
thanks closing via: https://github.com/HubSpot/offline/pull/251

**Pull request merged**